java/maven/README.md

```diff
@@ -6,6 +6,12 @@
 pack build applications/maven
 ```
 
+Alternatively, if you want to attach a Maven `settings.xml` file to pass additional configuration to Maven.
+
+```bash
+pack build applications/maven --volume (pwd)/bindings:/platform/bindings
```

Comment:
```suggestion
```bash
cp ~/.m2/settings.xml java/maven/binding/settings.xml
pack build applications/maven --volume (pwd)/bindings:/platform/bindings
```
@dmikusa-pivotal Do you think it worth suggesting folks copy in their local `settings.xml`? If they are trying to pull dependencies from a private artifactory for example, they may well already have the config they need in `~/.m2/settings.xml`.

Reply:
Yes, that's a good idea. I will add some text to that effect too. Like "the sample settings.xml file does nothing, but you can copy yours".